wasm/jsapi/instance/constructor-caching.any.js

```diff
@@ -0,0 +1,59 @@
+// META: global=jsshell
+// META: script=/wasm/jsapi/wasm-constants.js
+// META: script=/wasm/jsapi/wasm-module-builder.js
+
+function getExports() {
+  const builder = new WasmModuleBuilder();
+  builder
+    .addFunction("fn", kSig_v_d)
+    .addBody([])
+    .exportFunc();
+
+  builder.setTableLength(1);
+  builder.addExportOfKind("table", kExternalTable, 0);
+
+  builder.addGlobal(kWasmI32, false)
+    .exportAs("global")
+    .init = 7;
```

Comment:
The value isn't used, could we remove the assignement?

Reply:
Sure.